config.yml.template

```diff
@@ -5,6 +5,7 @@ email:
   imapUrl: imap.gmail.com
```

Comment:
we should make a note that the "password" field is deprecated for new installations

Reply:
Updated comments with password and removed references to imap/smtp/gmail urls that moved to constants